in-development/mission_3774_sap_task_center/step_2/variables.tf

```
@@ -0,0 +1,119 @@
+######################################################################
+# Customer account setup
+######################################################################
+variable "globalaccount" {
+  type        = string
+  description = "Defines the global account"
+  default     = "yourglobalaccount"
+}
+
+variable "cli_server_url" {
+  type        = string
+  description = "Defines the CLI server URL"
+  default     = "https://cli.btp.cloud.sap"
+}
+
+# subaccount
+variable "subaccount_name" {
+  type        = string
+  description = "The subaccount name."
+  default     = "UC - Establish a Central Inbox with SAP Task Center"
+}
+variable "subaccount_id" {
+  type        = string
+  description = "The subaccount ID."
+  default     = ""
+}
+# Region
+variable "region" {
+  type        = string
+  description = "The region where the project account shall be created in."
+  default     = "us10"
+}
+# Cloudfoundry environment label
+variable "cf_environment_label" {
+  type        = string
+  description = "The Cloudfoundry environment label"
+  default     = "cf-us10"
+}
+
+variable "subaccount_admins" {
+  type        = list(string)
+  description = "Defines the colleagues who are added to each subaccount as subaccount administrators."
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+variable "subaccount_service_admins" {
+  type        = list(string)
+  description = "Defines the colleagues who are added to each subaccount as subaccount service administrators."
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+variable "launchpad_admins" {
+  type        = list(string)
+  description = "Defines the colleagues who are added to each subaccount as subaccount service administrators."
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+variable "custom_idp" {
+  type        = string
+  description = "Defines the custom IdP"
+  default     = ""
+}
+
+variable "environment_label" {
+  type        = string
+  description = "In case there are multiple environments available for a subaccount, you can use this label to choose with which one you want to go. If nothing is given, we take by default the first available."
+  default     = "cf-us10"
+}
+
+variable "cf_org_name" {
+  type        = string
+  description = "Name of the Cloud Foundry org."
+  default     = "mission-3774-sap-task-center"
+
+  validation {
+    condition     = can(regex("^.{1,255}$", var.cf_org_name))
+    error_message = "The Cloud Foundry org name must not be emtpy and not exceed 255 characters."
+  }
+}
+
+variable "cf_space_name" {
+  type        = string
+  description = "Name of the Cloud Foundry space."
+  default     = "dev"
+}
+
+variable "cf_username" {
+  type        = string
+  description = "CF User Name"
+  default     = ""
+}
+
+variable "cf_password" {
+  type        = string
+  description = "CF Password"
+  default     = ""
+}
+
+variable "cfsr_space_manager" {
```

QAS naming convention 
```suggestion
variable "cf_space_manager" {
```